official/resnet/keras/keras_imagenet_main.py

```diff
@@ -117,6 +117,13 @@ def run_imagenet_with_keras(flags_obj):
   Args:
     flags_obj: An object containing parsed flag values.
   """
+  # Raise an error if the user wants to run with FP16 support. Currently we do
```

Comment:
nit.  comment not needed, the error message is self explanatory.  The Error message could also be shorter with something like:  "fp16 not supported in Keras. Use fp32 (default)."   If I wish I was better with punctuation.